### PR TITLE
Add initial bootstrap governance

### PR DESCRIPTION
**Pull request by afrittoli**
Add an initial governance.md to define the bootstrap governance
committee

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

### CODE REVIEW

File: governance.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+The initial bootstrap committee will consist of 5 individuals who are core stakeholders and/or contributors.
+
+Members are (in alphabetical order):
+
+* Emil Bäckmark, [@e-backmark-ericsson](https://github.com/e-backmark-ericsson), Ericsson
+* Andrea Frittoli, [@afrittoli](https://github.com/afrittoli), IBM
+* Mattias Linnér, [@m-linner-ericsson](https://github.com/m-linner-ericsson), Ericsson
+* Erik Sternerson [@erkist](https://github.com/erkist), doWhile
+* Steve Taylor [@sbtaylor15](https://github.com/sbtaylor15), [DeployHub](https://www.deployhub.com) / [Ortelius OS](https://ortelius.io)
```
Comment (afrittoli) [inline governance.md:11]: The list here is based on who more frequently (and recently) attended the sig-events vocabulary meeting.
Feel free to comment if you'd prefer not to be in this list or to be added to it - we can discuss about this in the next meeting too. We don't necessarily need to have 5 individuals either - this is what the Tekton project did and it worked well there.

File: governance.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+Members of the governance committee will be responsible for handling [CDEvents code of conduct](code-of-conduct.md)
+violations via cdevents-code-of-conduct@googlegroups.com.
+
+## Project Communication Channels
+
+What are the primary communications channels the project will adopt and manage?
+This can include Slack, mailing lists, an organized Stack Overflow topic, or exist only in GitHub issues and pull requests.
+
+* Mailing list: [groups.google.com/g/cdevents-dev](https://groups.google.com/g/cdevents-dev).
+* Slack: #sig-events [slack channel](https://cdeliveryfdn.slack.com/archives/C0151BTKEJX) on the CDF slack.
```
Comment (afrittoli) [inline governance.md:88]: These are two examples here, but we will end-up documenting our communication channels in the main README (and/or in community.md)

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
The title should be CD Events not Tekton

As discussed at the CD Events SIG, it might make sense to have this file in .github so that it can be referenced from all repositories.

File: governance.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+* Explains to potential contributors how/if they can add code to the repository/repositories
+* Documents Workflow and management of pull requests
+* Identifies who is authorized to commit or revert code
+* Identifies automation is required for normal operations
+* Defines how release decisions are made
+  * Who is authorized to release and when.
+  * Frequency limits
+* Defines the documentation process
+* Defines what CLA process is required and how it is programmatically enforced before code is merged
```
Comment (m-linner-ericsson) [inline governance.md:72]: I think we should spell out abbreviations such as CLA or link to some page describing it e.g. https://en.wikipedia.org/wiki/Contributor_License_Agreement

File: governance.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
```
Comment (m-linner-ericsson) [inline governance.md:73]: Should the bootstrap committee also set up a security process? What if someones finds a security issue, how is this reported and handled?

From the [Eiffel Bootstrap committee governance](https://github.com/eiffel-community/community/commit/63883403ebcf07af3b312af103a0579819d8409c#diff-b60c6a93e9f74ee52e71bf33781c2870b8395c8c5a34f00f61305343cb8d8447)

```suggestion
### Security/Vulnerability Reporting and Response Process
* Identify and document where vulnerability reporting can be done to the project
* Identify and document who is responsible for receiving vulnerability reports
* Document process responsible parties go through to triage and determine veracity of vulnerability
* Document process for facilitating fix, generating release update, and communicating vulnerability and fix to public
```

**Comment by m-linner-ericsson** [comment]
> As discussed at the CD Events SIG, it might make sense to have this file in .github so that it can be referenced from all repositories.

I don't know if it was @oleg-nenashev or @sbtaylor15 who mentioned this. 

When looking again at the `.github` repository I didn't find any good reason for putting the governance there. I know it's good for code of conduct and contributor guidelines as GitHub automatically links it, but I can't see any purpose with the governance. Am I missing something here?

**Comment by oleg-nenashev** [comment]
It is basically style topic. As you prefer. I just do not want to have governance being version along with the spec tags. Sorry moving it to a separate repo makes sense to me

**Comment by afrittoli** [comment]
> 



> It is basically style topic. As you prefer. I just do not want to have governance being version along with the spec tags. Sorry moving it to a separate repo makes sense to me

The governance docs are not really versioned, so I'm not sure I see the issue?

We could have a dedicated repo for community / governance type of documents, owned by the project governance team, but that seemed a bit of an overhead to just get started - I wanted to avoid spending too much time in managing a lot of repos :P 

I you feel strongly about using the `.github` repo for this, it works for me.

File: governance.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
+# Tekton Bootstrap Governance
```
[the remark this answers is not in the document]
Comment (afrittoli) [reply governance.md:1]: Heh, good catch, thank you!

File: governance.md (hunk 4 above)
Comment (afrittoli) [reply governance.md:73]: Sure, I'm happy to add that

File: governance.md (hunk 3 above)
Comment (afrittoli) [reply governance.md:72]: done

File: governance.md (hunk 4 above)
Comment (afrittoli) [reply governance.md:73]: done

**Comment by afrittoli** [comment]
I addressed all comments. 
I also made a PR to the `.github` repo: https://github.com/cdevents/.github/pull/1
We can decide next time we meet which of the two to merge, or if we shall create a new repo to host this.

**Review by m-linner-ericsson — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
Time to merge it? All the text looks reasonable to me. IMHO there are too many lists planned, maybe some of them could be moved to GitHub Issues or discussions. The Code of Conduct escalations handing could be also directly delegated to the CDF, but no strong opinion here

**Comment by e-backmark-ericsson** [comment]
Merging this as a base for our govenance model

**Pull request merged**